SingularityUI/app/components/common/modalButtons/NewDisabledActionModal.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { FetchDisabledActions, NewDisabledAction } from '../../../actions/api/disabledActions';
+
+import FormModal from '../modal/FormModal';
+
+const DISABLED_ACTION_TYPES = ['BOUNCE', 'DEPLOY', 'SCALE', 'REMOVE', 'DECOMMISSION'];
+
+import Utils from '../../../utils';
+
+class DeleteDisabledActionModal extends Component {
+  static propTypes = {
+    user: PropTypes.string,
+    newDisabledAction: PropTypes.func.isRequired
+  };
+
+  show() {
+    this.refs.newDisabledActionModal.show();
+  }
+
+  render() {
+    return (
+      <FormModal
+        ref="newDisabledActionModal"
+        name="New Disabled Action"
+        action="Create Disabled Action"
+        buttonStyle="success"
+        onConfirm={(data) => this.props.newDisabledAction(data.type, data.message)}
+        formElements={[
+          {
+            type: FormModal.INPUT_TYPES.SELECT,
+            name: 'type',
+            label: 'Type',
+            isRequired: true,
+            options: DISABLED_ACTION_TYPES.map((type) => ({
```

Comment:
It would be nice to exclude already-disabled actions from this list.
(It would also be nice to be able to select multiple. But I don't think we have a multiselect component in FormModal yet, and I don't know if the backend would even allow it.)